docs/core/diagnostics/metrics-instrumentation.md

```diff
@@ -376,12 +376,13 @@ has replaced the generic term "Count" that is visible in previous descriptions.
 
 ### Best practices
 
-- .NET APIs allow any string to be used as the unit, but we recommend using [UCUM](https://ucum.org/), an international standard for unit names. This is also what
-[OpenTelemetry naming guidelines](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md#instrument-units) recommend. The curly
+- .NET APIs allow any string to be used as the unit, but we recommend using [UCUM](https://ucum.org/), an international standard for unit names. The curly
 braces around "{hats}" is part of the UCUM standard indicating that it is a descriptive annotation rather than a unit name with a standardized meaning like seconds or meters.
```

Comment:
```suggestion
braces around "{hats}" is part of the UCUM standard, indicating that it is a descriptive annotation rather than a unit name with a standardized meaning like seconds or bytes.
```
To avoid confusion about the different meanings of meter.

Reply:
good call... I didn't even notice that.